overlay/base/usr/lib/system/_os/podman.py

```
@@ -751,3 +807,51 @@ def pull(
+    pretty: bool = False,
+) -> list[dict[str, Any]]:  # pyright: ignore[reportExplicitAny]
+    is_path = isinstance(containerfile, str)
+    if is_path:
+        containerfile = open(containerfile, "r")
+
+    argv = ["-p"] if pretty else []
+    if build_args is not None:
+        argv += [x for k, v in build_args.items() for x in ["-b", f"{k}={v}"]]
+
+    data = json.loads(  # pyright: ignore[reportAny]
+        subprocess.check_output(
```

Ruff (0.14.4)

`subprocess` call: check for execution of untrusted input

(S603)